### ISSUE TITLE
Lazy.nvim support, as it does not support rocks.

### ISSUE THREAD

**Issue by vsedov** [jghauser/papis.nvim#18]
As of the moment lazy.nvim does not support rocks, is there another way of installing this plugin ?

**Comment by jghauser** [comment]
I've been wondering about that! I was thinking about directly integrating `lyaml` as a git submodule, but I think that can't work, because `lyaml` has parts written in C that need to be compiled. I guess you could install the `lyaml` rock independently on your system and then fiddle with with neovim's `package.path` and `package.cpath` so that neovim recognises the rock. If you come up with something reasonable that I could do to support lazy.nvim (or that would allow me to read `.yaml` files without having to require a luarock), please do let me know!

**Comment by jghauser** [comment]
That's great! Would be great if you could get back to me after testing this for a while. I'll then update the installation instructions so that lazy.nvim users can install this too (at least until we find a less clutch way of doing this). Thanks!

**Comment by vsedov** [comment]
Update this works, but, adding it to package.path doesnt really work i had to do some even more scuffed work around for now . 

![image](https://user-images.githubusercontent.com/28804392/225883461-f5367c54-0fe3-4cd0-ab97-827393ed6088.png)


where i just moved those files to my config, that being said im pretty certain there is a better way for this. Regarding the yaml.so path, that you can directly add through luarocks command xd 


I have a even more scuffed idea of just copying those files from lyaml, and the yaml .so and just making another package.  Mainly because 

```

path_yaml = "/home/viv/.luarocks/share/lua/5.1/lyaml/"
-- add to path
package.path = package.path .. ";" .. path_yaml .. "?.lua"

```
This wont work  or any other solution i conjured up, but the only one that does, and ive tested papis, works fine is adding it to yer config xd. 


On that note, while messing around with my papers, papis seemed to work without any errors.

**Comment by jghauser** [comment]
I've just checked in my local installation how the paths are set up. This is on nixos, so things look a bit different, but maybe this is helpful for you. The relevant paths seem to be:

- package.path `/nix/store/4ky29380a282m5iwrw3ma9wslih5h407-luajit2.1-lyaml-6.2.7-1/share/lua/5.1/?.lua;/nix/store/4ky29380a282m5iwrw3ma9wslih5h407-luajit2.1-lyaml-6.2.7-1/share/lua/5.1/?/init.lua`
- package.cpath: `/nix/store/4ky29380a282m5iwrw3ma9wslih5h407-luajit2.1-lyaml-6.2.7-1/lib/lua/5.1/?.so`

Maybe this is helpful to avoid having to move files around.

**Comment by jghauser** [comment]
This is now solved by #19
